### ISSUE TITLE
Another Skybell task exception error

### ISSUE THREAD

**Issue by GaryOkie** [MisterWil/skybellpy#4]
Hi MisterWil,

Apparently the latest Hassio update (.67) is causing another task exception with a distutils dependency.   Would you please reference issue [#13922](https://github.com/home-assistant/home-assistant/issues/13922) for further details?

I'm hoping this is a minor tweak.  I would be happy to assist with any testing.

**Comment by MisterWil** [comment]
Just upgraded my own HASS version yesterday and also noticed this issue. I'll get it fixed here and submit a PR to HASS to bump the version ASAP.  :+1:

**Comment by GaryOkie** [comment]
awesome response - thanks for taking care of it so quickly!

**Comment by MisterWil** [comment]
Released a version 0.1.2 to try and fix this. If you get a chance please test it!

**Comment by GaryOkie** [comment]
This will be my first attempt at installing a custom component for testing within HASSIO, and so far, it's not going well.     I created a zip file from your skybell repo, and uploaded all of this to a /config/custom_components/camera folder.    restarted hassio and get:

```
2018-04-17 10:32:16 ERROR (MainThread) [homeassistant.setup] Error during setup of component camera
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/homeassistant/setup.py", line 145, in _async_setup_component
    component.setup, hass, processed_config)
AttributeError: module 'custom_components.camera' has no attribute 'setup'
2018-04-17 10:32:24 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity_platform.py", line 260, in _async_add_entity
    await entity.async_update_ha_state()
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 215, in async_update_ha_state
    state = self.state
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 379, in state
    return STATE_ON if self.is_on else STATE_OFF
  File "/usr/lib/python3.6/site-packages/homeassistant/components/switch/skybell.py", line 75, in is_on
    return getattr(self._device, self._switch_type)
  File "/usr/lib/python3.6/site-packages/skybellpy/device.py", line 206, in do_not_disturb
    return bool(distutils.util.strtobool(str(self._settings_json.get(
AttributeError: module 'distutils' has no attribute 'util'
2018-04-17 10:32:57 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 215, in async_update_ha_state
    state = self.state
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 379, in state
    return STATE_ON if self.is_on else STATE_OFF
  File "/usr/lib/python3.6/site-packages/homeassistant/components/switch/skybell.py", line 75, in is_on
    return getattr(self._device, self._switch_type)
  File "/usr/lib/python3.6/site-packages/skybellpy/device.py", line 206, in do_not_disturb
    return bool(distutils.util.strtobool(str(self._settings_json.get(
AttributeError: module 'distutils' has no attribute 'util'
2018-04-17 10:33:28 ERROR (MainThread) [homeassistant.core] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 215, in async_update_ha_state
    state = self.state
  File "/usr/lib/python3.6/site-packages/homeassistant/helpers/entity.py", line 379, in state
    return STATE_ON if self.is_on else STATE_OFF
  File "/usr/lib/python3.6/site-packages/homeassistant/components/switch/skybell.py", line 75, in is_on
    return getattr(self._device, self._switch_type)
  File "/usr/lib/python3.6/site-packages/skybellpy/device.py", line 206, in do_not_disturb
    return bool(distutils.util.strtobool(str(self._settings_json.get(
AttributeError: module 'distutils' has no attribute 'util'
```

**Comment by GaryOkie** [comment]
I wasn't sure if "camera" was the right subfolder name, so renamed it to skybell and get:

```
2018-04-17 10:51:24 ERROR (MainThread) [homeassistant.setup] Error during setup of component skybell
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/homeassistant/setup.py", line 145, in _async_setup_component
    component.setup, hass, processed_config)
AttributeError: module 'custom_components.skybell' has no attribute 'setup'
2018-04-17 10:51:26 ERROR (MainThread) [homeassistant.setup] Unable to setup dependencies of camera.skybell. Setup failed
2018-04-17 10:51:26 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform camera.skybell: Could n
2018-04-17 10:51:26 ERROR (MainThread) [homeassistant.setup] Unable to setup dependencies of switch.skybell. Setup failed
2018-04-17 10:51:26 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform switch.skybell: Could n
2018-04-17 10:54:36 ERROR (MainThread) [homeassistant.core] Error doing job: Task was destroyed but it is pending!

```

**Comment by MisterWil** [comment]
I honestly have no idea how to upgrade libraries in hassio. Usually I just modify the component directly in /usr/lib/python3.6/site-packages/homeassistant/components/skybell.py, change the requirement version near the top, and then restart. Usually this forces the library to upgrade.

**Comment by GaryOkie** [comment]
My apologies for not having any experience with custom updates to solve this on my own.  

So, I assume you've tested it and found that it solved the distutil error, right?

**Comment by MisterWil** [comment]
@GaryOkie Yeah I tested it last night and I'm pretty sure the error went away.

@dale3h I noticed you tried to fix this as well. Are you able to easily update skybellpy to 0.1.2 in HASS and test as well?

**Comment by MisterWil** [comment]
I believe I've now fixed this in 0.1.2 so I'm closing this. Submitted a PR to HASS to pull in the new version.
